### PR TITLE
editor_support.md: Remove headings

### PR DESCRIPTION
**Pull request by RohanVB**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by RohanVB** [comment]
Not sure if this would make the readability better @sims1253 

> looking at this: http://projects.coala.io/#/projects?project=cohesive_editor_support could you make the detail blocks indented or something so they look different from the text?

File: data/projects/editor_support.md [hunk 1]
```diff
@@ -14,36 +14,36 @@ widely used, but it would also benefit the end-users by giving them a working
...
-Details (for documentation):
-
-Documentation which isn't available at the moment will have to describe the 
-usage details of existing plugins, we can have asciicinemas or gifs of the
-entire process from installation to examples of the plugins in action.
-Documentation can also be created for future plugins, and on the basis of the
-tests which will be written for the existing ones, it'll be easier to test
-the future plugins. This will result in a clear cut path of creating plugins
-for coala.
+ Details (for testing):
```
Comment (Adrianzatreanu) [inline data/projects/editor_support.md]: im clicking "View" in github and the indentation still doesnt work :/

Comment (RohanVB) [inline data/projects/editor_support.md]: gotta add them `&nbsp;&nbsp;`'s :D

**Comment by Adrianzatreanu** [comment]
this looks horrible to be honest

**Comment by RohanVB** [comment]
I know :(

**Comment by adhikasp** [comment]
How about using blockquote?

**Comment by sims1253** [comment]
you could also remove the mini headers and just make it paragraphs then there is no real need for indentation

**Comment by RohanVB** [comment]
I kind of wanted to highlight the different descriptions we have for the big objectives of the project, but this looks alright too.

**Comment by arnab1896** [comment]
@RohanVB , @sims1253 , according to what was being discussed on gitter the other day;
Aren't we supposed to name the three phases as:-
"GSOC 2017 CODING PHASE 1"
"GSOC 2017 CODING PHASE 2"
"GSOC 2017 CODING PHASE 3"
instead of :- 
"CODING PHASE 1"
"CODING PHASE 2"
"CODING PHASE 3"

Let me know if i'm wrong;

Cheers!!

**Comment by arnab1896** [comment]
@RohanVB ; i was wrong. it's fine as it is right now.
My bad!

**Comment by supergr35** [comment]
ack fde5952

**Comment by aptrishu** [comment]
rebase needed..

**Comment by sims1253** [comment]
ack fde5952

**Comment by RohanVB** [comment]
reack fde5952

**Comment by RohanVB** [comment]
reack eb7cbb9a69bea447aae3b91147f7213aa53f8cc1

**Comment by RohanVB** [comment]
reack eb7cbb9a69bea447aae3b91147f7213aa53f8cc1

**Pull request merged**